### ISSUE TITLE
Can you add a scroll bar component or beautify the default scroll bar?

### ISSUE THREAD

**Issue by Lan-W** [ant-design/ant-design#24674]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
When making some big data platforms, designers often design dark themes. The default scroll bar of browsers is too eye-catching. When they introduce a third-party scroll bar beautification plug-in, they can't set the table or select components. 

### What does the proposed API look like?
<a-scrollbar>
...
</a-scrollbar>

**Comment by afc163** [comment]
#12381

I suggest using https://css-tricks.com/custom-scrollbars-in-webkit/

**Comment by Lan-W** [comment]
Thank you. I've tried this method. It's really useful on browsers with WebKit as the kernel, but it's not compatible with IE and Firefox. At the same time, I've tried to use the 'overlayscrollbars' plug-in. General scrolling and table components have solutions, but they can't be used in the scroll bar of select, which bothers me. So I'm looking for your help to see if we can implement the scroll bar like elementui, Or there are other solutions.
